src/renderer/features/network/NetworkSelector/model/network-selector-model.ts

```diff
@@ -0,0 +1,133 @@
+import { createEvent, sample, createEffect, attach } from 'effector';
+import { delay, spread } from 'patronum';
+
+import { ProviderWithMetadata } from '@/src/renderer/shared/api/network';
+import { networkModel, networkUtils } from '@entities/network';
+import { ChainId, ConnectionType, RpcNode } from '@shared/core';
+
+const lightClientSelected = createEvent<ChainId>();
+const autoBalanceSelected = createEvent<ChainId>();
+const rpcNodeSelected = createEvent<{ chainId: ChainId; node: RpcNode }>();
+const chainDisabled = createEvent<ChainId>();
+
+const updateConnectionFx = networkModel.effects.updateConnectionFx;
```

Comment:
If `networkModel.effects.updateConnectionFx` is never used anywhere else, it can be moved into this model
Otherwise you should use `attach` to wrap the initial effect
It's needed to have an isolated copy of the effect, so anybody who tracks original `updateConnectionFx` won't trigger accidentally. [doc](https://effector.dev/en/api/effector/attach/#methods-attach-effect-examples)